src/components/table/table.stories.js

```diff
@@ -26,12 +26,12 @@ const handleChange = (e, tableOptions) => {
   action('change')(e, tableOptions);
 };
 
-const buildRows = (pageSizeFromKnobs) => {
+const buildRows = (pageSizeFromKnobs, totalRecords) => {
   const pageSize = pageSizeFromKnobs;
   const currentPage = store.get('currentPage');
 
-  const endIndex = pageSize * currentPage;
-  const startIndex = endIndex - pageSize;
+  const endIndex = pageSize * currentPage <= totalRecords ? pageSize * currentPage : totalRecords;
```

Comment:
Could create a variable for `pageSize * currentPage`

Reply:
Yeah I've gone and added it